docs/recipes/react/example-effects.md

```diff
@@ -37,4 +37,4 @@ const App = () => {
 ReactDOM.render(<App />, document.getElementById('root'))
 ```
 
-[Try it](https://share.effector.dev/wdI27XSm)
\ No newline at end of file
+[Try it](https://share.effector.dev/wdI27XSm)
```

Comment:
As I see this REPL still has an error

![image](https://user-images.githubusercontent.com/26767722/162656493-d6427d0a-e5ad-4a52-97ba-53c62c93d49f.jpeg)

Reply:
I mean, any REPL is immutable. You need to change source of it and attach new link to the md file.